operators/multiclusterobservability/api/v1beta2/multiclusterobservability_types.go

```diff
@@ -116,6 +140,12 @@ type PlatformCapabilitiesSpec struct {
 	// +optional
 	// +kubebuilder:validation:Optional
 	Metrics PlatformMetricsSpec `json:"metrics,omitempty"`
+	// Analytics defines the spec for the analytics capabilities.
+	// Currently it support Right Sizing recommendation feature.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	Analytics *AnalyticCapabilitiesSpec `json:"analytics,omitempty"`
```

Comment:
nit: I would probably call it only `AnalyticsSpec` and would remove the:
>// Currently it support Right Sizing recommendation feature.

because it won't be excatly true once there's also the incident detection included

Reply:
Merged changes with latest as changes are already in place with incident detection PR.